### PR TITLE
Added constants to sqldwadaptor to set connection timeout, request timeout and concurrent request limit

### PR DESCRIPTION
**Pull request by mantmic**

### CODE REVIEW

**Comment by mantmic** [comment]
@BenBirt

File: api/dbadapters/sqldatawarehouse.ts [hunk 1]
```diff
@@ -11,6 +11,10 @@ const TABLE_TYPE_COL_NAME = "table_type";
 const COLUMN_NAME_COL_NAME = "column_name";
 const DATA_TYPE_COL_NAME = "data_type";
 const IS_NULLABLE_COL_NAME = "is_nullable";
+const DB_CONNECTION_TIMEOUT = 300000; //5 minute connection timeout
```
Comment (BenBirt) [inline api/dbadapters/sqldatawarehouse.ts:14]:
```suggestion
const DB_CONNECTION_TIMEOUT_MILLIS = 5 * 60 * 1000; //5 minute connection timeout
```

(similar suggestion below)

Comment (BenBirt) [inline api/dbadapters/sqldatawarehouse.ts:14]: supernit: space after `//` please (same below)

File: api/dbadapters/sqldatawarehouse.ts [hunk 2]
```diff
@@ -28,6 +32,12 @@ export class SQLDataWarehouseDBAdapter implements IDbAdapter {
         user: sqlDataWarehouseCredentials.username,
         password: sqlDataWarehouseCredentials.password,
         database: sqlDataWarehouseCredentials.database,
+        connectionTimeout: DB_CONNECTION_TIMEOUT,
+        requestTimeout: DB_REQUEST_TIMEOUT,
+        pool:{
```
Comment (BenBirt) [inline api/dbadapters/sqldatawarehouse.ts:37]:
```suggestion
        pool: {
```

**Pull request merged**